src/sentry/api/serializers/models/project_key.py

```diff
@@ -100,4 +103,8 @@ def serialize(
                 "hasDebug": get_dynamic_sdk_loader_option(obj, DynamicSdkLoaderOption.HAS_DEBUG),
             },
         }
+
+        if user.is_superuser:
```

Comment:
Superuser is being split into two roles (staff and superuser). Staff role users will have access to internal tools like `_admin` but not to customer data. How does superuser create these internal DSNs?

Reply:
The intention here is that sentry developers can view the internal DSNs of a project, but the customers themselves cannot. These internal DSNs are not really customer data, but they're associated to a customer project. Would you say `staff` would be the more appropriate role for this?

Reply:
In that case, the check can remain as superuser. Mark also mentioned this but to check for active superuser you can use `is_active_superuser`.

`user.superuser` is only checking that a user can become an active superuser, not that they actually are one.

Reply:
@schew2381 thanks! Replaced with `is_active_superuser` check now.